library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/impl/BlockContentRegistriesClientInitializer.java

```diff
@@ -35,7 +35,7 @@ public class BlockContentRegistriesClientInitializer implements ClientModInitial
 
 	@Override
 	public void onInitializeClient(ModContainer mod) {
-		if (Boolean.getBoolean(ENABLE_TOOLTIP_DEBUG)) {
+		if (Boolean.getBoolean(ENABLE_TOOLTIP_DEBUG) || QuiltLoader.isModLoaded("quilt_block_content_registry_testmod")) {
```

Comment:
I really have been needing that! Although yeah, we really should investigate having a debug config file for this kind of stuff

Reply:
Yeah that is something i want to look into later, but not sure the best way to approach. Maybe a QSL testmod api or something. As for this, i think i will leave it in because it is *extremely* useful when testing